README.md

```diff
@@ -10,204 +21,42 @@
 | **Social** | <a href="https://foundrydigital.com/accelerate/" target="_blank"> <img src="https://img.shields.io/website?url=https%3A%2F%2Ffoundrydigital.com%2Faccelerate%2F&up_message=Foundry%20Accelerate&label=Website" height="25"/> </a> <br> <a href="https://taostats.io/validators/5HEo565WAy4Dbq3Sv271SAi7syBSofyfhhwRNjFNSM2gP9M2" target="_blank"> <img src="https://img.shields.io/website?url=https%3A%2F%2Ftaostats.io%2Fvalidators%2Ffoundry%2F&up_message=Foundry%20Accelerate&label=Validator" height="25"/> </a> <br> <a href="https://x.com/FoundryServices?s=20" target="_blank"> <img src="https://img.shields.io/twitter/follow/FoundryServices" height="25"/> </a> |
 
-
 </div>
 
----
-
-- [Introduction](#introduction)
-- [Design Decisions](#design-decisions)
-- [Installation](#installation)
-  - [Install PM2](#install-pm2)
-  - [Compute Requirements](#compute-requirements)
-  - [Install-Repo](#install-repo)
-  - [Running a Miner](#running-a-miner)
-  - [Running a Validator](#running-a-validator)
-    - [Obtain \& Setup WandB API Key](#obtain--setup-wandb-api-key)
-  - [Running Miner/Validator in Docker](#running-minervalidator-in-docker)
-- [About the Rewards Mechanism](#about-the-rewards-mechanism)
-  - [Miner Ranking](#miner-ranking)
-  - [$\\Delta$ Calculation for ranking miners](#delta-calculation-for-ranking-miners)
-  - [Exponential Decay Weighting](#exponential-decay-weighting)
-- [Roadmap](#roadmap)
-- [License](#license)
-
----
 ## Introduction
 
-Foundry is launching Foundry S&P 500 Oracle to incentivize miners to make predictions on the S&P 500 price frequently throughout trading hours. Validators send miners a timestamp (a future time), which the miners need to use to make predictions on the close price of the S&P 500 for the next six 5m intervals. Miners need to respond with their prediction for the price of the S&P 500 at the given time. Validators store the miner predictions, and then calculate the scores of the miners after the predictions mature. Miners are ranked against eachother, naturally incentivizing competition between the miners.
+Foundry Digital is launching the Foundry S&P 500 Oracle. This subnet incentivizes accurate short term price forecasts of the S&P 500 during market trading hours.
+
+Miners use Neural Network model architectures to perform short term price predictions on the S&P 500.
+
+Validators store price forecasts for the S&P 500 and compare these predictions against the true price of the S&P 500 as the predictions mature.
+
+## Usage
+
+<div align="center">
+
+| Miners | Validators |
+| :----: | :--------: |
+| [TestNet Docs]() | [TestNet Docs]() |
```

Comment:
Testnet, Mainnet. Or testnet, mainnet. I have not seen this capitalization before.